src/datasource/components/Logs/LogsCell.tsx

```diff
@@ -115,11 +119,13 @@ const ExpandedDocument  = ({ log, index, datasourceUid, datasourceName, datasour
                 </tr>
                 {
                     Array.from(log.keys()).map((key) => (
-                        <ExpandedLogKeyVal
-                            field={key}
-                            val={log.get(key)}
-                            key={key}
-                        />
+                        key !== '_source' ?
```

Comment:
`_source` should probably not be hardcoded, and come from the config. 

![image](https://github.com/slackhq/slack-kaldb-app/assets/771133/23cce645-0003-4b47-8cc6-4334e20cbe65)